Sources/TokenizersWrapper/TokenizersWrapper.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+import Hub
+
+@_exported import TokenizersCore
```

Comment:
This is a good chunk of the magic. The new `Tokenizers` implementation is just this wrapper file, which exposes the imported `TokenizersCore` types.